lib/Backend/Lower.cpp

```diff
@@ -10687,28 +10664,24 @@ Lowerer::LowerBailOnNotStackArgs(IR::Instr * instr)
         instr->InsertAfter(continueLabelInstr);
     }
 
-    IR::LabelInstr * helperLabelInstr = IR::LabelInstr::New(Js::OpCode::Label, m_func, true);
     if (!instr->m_func->IsInlinee())
     {
-        //BailOut if it is not stack args or the number of actuals (except "this" argument) is greater than or equal to 15.
-        IR::Opnd* stackArgs = instr->UnlinkSrc1();
-        InsertCompareBranch(stackArgs, instr->UnlinkSrc2(), Js::OpCode::BrNeq_A, helperLabelInstr, instr);
-
+        //BailOut if the number of actuals (except "this" argument) is greater than or equal to 15.
         IR::RegOpnd* ldLenDstOpnd = IR::RegOpnd::New(TyUint32, instr->m_func);
-        IR::Instr* ldLen = IR::Instr::New(Js::OpCode::LdLen_A, ldLenDstOpnd, stackArgs, instr->m_func);
+        IR::Instr* ldLen = IR::Instr::New(Js::OpCode::LdLen_A, ldLenDstOpnd, instr->m_func);
```

Comment:
Never mind. I didn't see the call to GenerateFastRealStackArgumentsLdLen below.

---
In reply to: [69179026](https://github.com/Microsoft/ChakraCore/pull/1211#discussion_r69179026) [](ancestors = 69179026)